simpeg_drivers/utils/synthetics/surveys/natural_sources/magnetotellurics.py

```diff
@@ -0,0 +1,47 @@
+# '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2025 Mira Geoscience Ltd.                                          '
+#                                                                                   '
+#  This file is part of simpeg-drivers package.                                     '
+#                                                                                   '
+#  simpeg-drivers is distributed under the terms and conditions of the MIT License  '
+#  (see LICENSE file at the root of this source code package).                      '
+#                                                                                   '
+# '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+import numpy as np
+from geoh5py import Workspace
+from geoh5py.objects.surveys.electromagnetics.magnetotellurics import MTReceivers
+
+
+channels = [
```

Comment:
```suggestion
CHANNELS = [
```
Constants should be uppercased. Would actually be better to be default input of the function.

Reply:
removed in favour of defaulted channels arguments